app/tests/cases_tests/test_nifti.py

```diff
@@ -0,0 +1,47 @@
+import os
+import shutil
+from pathlib import Path
+
+import pytest
+
+from grandchallenge.cases.image_builders.nifti import image_builder_nifti
+from grandchallenge.cases.models import Image
+from tests.cases_tests import RESOURCE_PATH
+
+
+def float_close(f1: float, f2: float) -> bool:
+    return abs(f1 - f2) < 0.0001
```

Comment:
Please use `pytest.approx` https://docs.pytest.org/en/5.4.1/reference.html#pytest-approx

Reply:
Neat, didn't know about this!

Reply:
No problem, this is what code review is for!